account_invoice_tax_required/models/account_move.py

```diff
@@ -44,6 +44,7 @@ def action_post(self):
                 config["test_enable"],
             )
         )
-        if self.type != "entry" and (force_test or not skip_test):
-            self._test_invoice_line_tax()
+        for record in self:
+            if record.type != "entry" and (force_test or not skip_test):
```

Comment:
This should be `if record.is_invoice()` (not your fault, but it's not correct either).

Reply:
Ok, thanks! I also changed the if condition to avoid having to check `force_test or not skip_test` for each iteration, when it can be done at the start.